prep-javascript-conditionals/index.html

```diff
@@ -13,27 +13,27 @@
     if (age < 18) {
       text = 'A child';
     } else { text = 'An adult'; }
-    console.log('An adult');
+    console.log('An adult', age);
 
     const student1Score = 99;
     if (student1Score >= 70) {
       text = 'Congrats you received a passing grade!';
     } else { text = 'Oh no you failed!'; }
-    console.log('Congrats you received a passing grade!');
+    console.log('Congrats you received a passing grade!', student1Score);
 
     const student2Score = 55;
     if (student2Score < 60) {
       text = 'F';
     } else if (student2Score < 70) { text = 'D'; } else if (student2Score < 80) { text = 'C'; } else if (student2Score < 90) { text = 'B'; } else if (student2Score < 100) { text = 'A'; } else { text = 'A++'; }
```

Comment:
Hi Nadia, for this exercise, log your `text` variable instead of the variables you are using in your conditions. We want the program to output values based on the conditional logic.